src/pages/Champions.js

```diff
@@ -0,0 +1,867 @@
+import React, { Component } from 'react';
+
+import pngTraits from '../components/pngTraits'
+import pngChampions from '../components/pngChampions'
+import championData from '../components/apilol/champions.json'
+
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route,
+    Link,
+    useParams
+  } from "react-router-dom";
+  
+class Champions extends React.Component {
+    //champsData
+    // champId = data.map( (data) =>{
+    //         return (data.champId)
+    //     }
+    // )
+    name = championData.map( (data) =>{
+            return (data.name)
+        }
+    )
+    // name = championInfo.map( (data) =>{
+    //         return (data.name.replace(/ /g,''))
+    //     }
+    // )
+    cost = championData.map( (data) =>{
+            return (data.cost)
+        }
+    )
+    traits = championData.map( (data) =>{
+            if(data.traits[2] == undefined){            //All the champs have the 2 synergy some has a 3 synergy
+                return  (data.traits[0] + ',\n' + data.traits[1]);
+            } 
+            else{
+                return (data.traits[0] + ',\n' + data.traits[1] + ',\n' + data.traits[2])
+            }
+        }
+    )
+    render() {
+        console.log(this.name)
+        // pngChampions
+        const pngChampion = pngChampions.map( (pngChampions) =>{
+                return (pngChampions.name)
+            }
+        )
+        // pngTraits
+        const pngTrait = pngTraits.map( (pngTraits) =>{
+                return (pngTraits.trait)
+            }
+        )
+        function ID() {
+            let { id } = useParams();
+            return (
+                <h1>ID: {id}</h1>
+            );
+        }
+        return (
+            <div>
+                <Router>
+                    <div>
+                        <ul>
+                            <li>
+                                <Link to="/ahri">{this.name[0]}</Link> {/* Ahri */}
+                            </li>
+                            <li>
+                                <Link to="/annie">{this.name[1]}</Link> {/* Annie */}
+                            </li>
+                            <li>
+                                <Link to="/ashe">{this.name[2]}</Link> {/* Ashe */}
+                            </li>
+                            <li>
+                                <Link to="/aurelionsol">{this.name[3]}</Link> {/* Aurelionsol */}
+                            </li>
```

Comment:
You can generate html with loops.

```suggestion
 { this.names.map((name) => {
    return (
         <li>
           <Link to="/ahri">{name}</Link>
         </li>
       )
  }
 }
```
You can resolve the issue with `link` by simply adding the href to their objects.